easybuild/easyconfigs/d/detettore/detettore-2.0.2-foss-2021a-Python-3.9.5.eb

```diff
@@ -0,0 +1,42 @@
+# This file is an EasyBuild reciPY as per https://github.com/hpcugent/easybuild
+# Author: Iñaki Mtz de Ilarduya Muñoz
+# sciCORE - University of Basel
+# SIB Swiss Institute of Bioinformatics
+
+easyblock = 'PythonPackage'
+
+name = 'detettore'
+version = '2.0.2'
+versionsuffix = '-Python-%(pyver)s'
```

Comment:
@i-mtz For Python packages that are installed with a recent toolchain, we're not longer using the `-Python-*` versionsuffix as long as Python 3 is used (we're still doing it when using Python 2).

So please remove this line, and rename the easyconfig file accordingly.